src/util/oneshot_holder.rs

```diff
@@ -0,0 +1,155 @@
+//
+// Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+
+use std::collections::vec_deque::Drain;
+use std::collections::VecDeque;
+use tokio::sync::oneshot::error::RecvError;
+use tokio::sync::oneshot::Receiver;
+
+pub struct OneShotHolder<E> {
+    size: usize,
+    inflight: VecDeque<tokio::sync::oneshot::Receiver<Result<(), E>>>,
```

Comment:
Receiver is imported. No need to specify the full path.

Reply:
removed the full path.